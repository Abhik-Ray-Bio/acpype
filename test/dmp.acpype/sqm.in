Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6    C1        1.3900       -5.8510       -5.5260 
   6   C20        0.4280       -5.6090       -7.6450 
   6   C21       -0.6890       -4.8550       -8.2410 
   6   C22       -0.8680       -4.8190       -9.6150 
   6   C23       -1.9300       -4.1120      -10.1540 
   6   C24       -2.8100       -3.4430       -9.3180 
   6   C25       -2.6300       -3.4800       -7.9500 
   6   C26       -1.5740       -4.1860       -7.4100 
   6   C27       -3.9620       -2.6770       -9.8910 
   6    C3        1.5440       -3.5850       -6.5890 
   6   C30        3.0460       -3.6240       -6.8590 
   6   C31        3.3350       -3.9930       -8.2840 
   6   C32        4.2460       -5.0040       -8.5420 
   6   C33        4.5100       -5.3760       -9.8420 
   6   C34        3.8670       -4.7400      -10.8850 
   6   C35        2.9550       -3.7260      -10.6300 
   6   C36        2.6860       -3.3540       -9.3240 
   6    C4        1.0520       -2.6460       -5.4870 
   6    C5        1.5830       -2.9550       -4.1090 
   6    C6        1.1400       -4.3060       -3.5610 
   6   C60       -0.3600       -4.5090       -3.3780 
   6   C61       -0.6930       -5.4970       -2.3050 
   6   C62       -1.5210       -6.5460       -2.6440 
   6   C63       -1.8530       -7.5000       -1.7120 
   6   C64       -1.3590       -7.4050       -0.4290 
   6   C65       -0.5250       -6.3490       -0.0770 
   6   C66       -0.1880       -5.3880       -1.0220 
   6   C70        2.3450       -6.5350       -3.5000 
   6   C71        3.4260       -6.0790       -2.6060 
   6   C72        3.7060       -6.7690       -1.4390 
   6   C73        4.7300       -6.3380       -0.6100 
   6   C74        5.4680       -5.2160       -0.9480 
   6   C75        5.1870       -4.5270       -2.1140 
   6   C76        4.1680       -4.9560       -2.9420 
   6   C77        6.5810       -4.7490       -0.0610 
   1  HH20        0.0660       -6.5820       -7.3390 
   1    H1        1.1720       -5.7590       -8.4130 
   1   H22       -0.1830       -5.3390      -10.2650 
   1   H23       -2.0720       -4.0810      -11.2250 
   1   H25       -3.3190       -2.9580       -7.3010 
   1   H26       -1.4360       -4.2130       -6.3370 
   1  HH27       -4.5720       -2.2990       -9.0790 
   1    H8       -4.5570       -3.3290      -10.5070 
   1    H3        1.0960       -3.2030       -7.4960 
   1  HH30        3.5040       -4.3520       -6.2050 
   1    H9        3.4690       -2.6520       -6.6560 
   1   H32        4.7520       -5.5010       -7.7230 
   1   H33        5.2210       -6.1640      -10.0440 
   1   H34        4.0740       -5.0370      -11.8960 
   1   H35        2.4540       -3.2300      -11.4490 
   1   H36        1.9750       -2.5640       -9.1190 
   1    H4       -0.0260       -2.7100       -5.4500 
   1    H5        2.6640       -2.9460       -4.1560 
   1    H6        1.5640       -4.3580       -2.5670 
   1  HH60       -0.7760       -4.8540       -4.3140 
   1   H10       -0.8110       -3.5600       -3.1280 
   1   H62       -1.9070       -6.6170       -3.6520 
   1   H63       -2.5010       -8.3190       -1.9840 
   1   H64       -1.6190       -8.1550        0.2910 
   1   H65       -0.1400       -6.2760        0.9290 
   1   H66        0.4600       -4.5620       -0.7580 
   1  HH70        2.7510       -7.2550       -4.1990 
   1   H11        1.6110       -7.0410       -2.8950 
   1   H72        3.1300       -7.6400       -1.1760 
   1   H73        4.9500       -6.8750        0.3010 
   1   H75        5.7640       -3.6520       -2.3750 
   1   H76        3.9510       -4.4140       -3.8540 
   1  HH77        7.1560       -3.9960       -0.5860 
   1   H12        7.2210       -5.5810        0.1780 
   1  HHO2       -4.2200       -1.2400      -11.1740 
   1   HO4        0.7540       -1.0590       -6.5080 
   1   HO5        1.7120       -2.0870       -2.4090 
   1  HHO7        5.6000       -4.9090        1.6140 
   7    N2        1.0850       -4.9920       -6.4930 
   7    N7        1.6610       -5.5010       -4.2700 
   8    O1        1.4170       -7.0440       -5.8040 
   8   O27       -3.4810       -1.5960      -10.6770 
   8    O4        1.4010       -1.3640       -5.8670 
   8    O5        1.1760       -1.9950       -3.2010 
   8   O77        6.0510       -4.2050        1.1410 

