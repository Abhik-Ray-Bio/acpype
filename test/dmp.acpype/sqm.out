            --------------------------------------------------------
                             AMBER SQM VERSION 17
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =109

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      206
| QMMM:    diag iterations used for timing =        4
| QMMM:
| QMMM:              Internal diag routine = 0.107248 seconds
| QMMM:                 Dspev diag routine = 0.144074 seconds
| QMMM:                Dspevd diag routine = 0.091428 seconds
| QMMM:                Dspevx diag routine = 0.289472 seconds
| QMMM:                 Dsyev diag routine = 0.143060 seconds
| QMMM:                Dsyevd diag routine = 0.099155 seconds
| QMMM:                Dsyevr diag routine = 0.089959 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.070032 seconds
| QMMM:
| QMMM: Using dsyevr routine (diag_routine=7).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.3900   -5.8510   -5.5260
  QMMM:     2        2      C        0.4280   -5.6090   -7.6450
  QMMM:     3        3      C       -0.6890   -4.8550   -8.2410
  QMMM:     4        4      C       -0.8680   -4.8190   -9.6150
  QMMM:     5        5      C       -1.9300   -4.1120  -10.1540
  QMMM:     6        6      C       -2.8100   -3.4430   -9.3180
  QMMM:     7        7      C       -2.6300   -3.4800   -7.9500
  QMMM:     8        8      C       -1.5740   -4.1860   -7.4100
  QMMM:     9        9      C       -3.9620   -2.6770   -9.8910
  QMMM:    10       10      C        1.5440   -3.5850   -6.5890
  QMMM:    11       11      C        3.0460   -3.6240   -6.8590
  QMMM:    12       12      C        3.3350   -3.9930   -8.2840
  QMMM:    13       13      C        4.2460   -5.0040   -8.5420
  QMMM:    14       14      C        4.5100   -5.3760   -9.8420
  QMMM:    15       15      C        3.8670   -4.7400  -10.8850
  QMMM:    16       16      C        2.9550   -3.7260  -10.6300
  QMMM:    17       17      C        2.6860   -3.3540   -9.3240
  QMMM:    18       18      C        1.0520   -2.6460   -5.4870
  QMMM:    19       19      C        1.5830   -2.9550   -4.1090
  QMMM:    20       20      C        1.1400   -4.3060   -3.5610
  QMMM:    21       21      C       -0.3600   -4.5090   -3.3780
  QMMM:    22       22      C       -0.6930   -5.4970   -2.3050
  QMMM:    23       23      C       -1.5210   -6.5460   -2.6440
  QMMM:    24       24      C       -1.8530   -7.5000   -1.7120
  QMMM:    25       25      C       -1.3590   -7.4050   -0.4290
  QMMM:    26       26      C       -0.5250   -6.3490   -0.0770
  QMMM:    27       27      C       -0.1880   -5.3880   -1.0220
  QMMM:    28       28      C        2.3450   -6.5350   -3.5000
  QMMM:    29       29      C        3.4260   -6.0790   -2.6060
  QMMM:    30       30      C        3.7060   -6.7690   -1.4390
  QMMM:    31       31      C        4.7300   -6.3380   -0.6100
  QMMM:    32       32      C        5.4680   -5.2160   -0.9480
  QMMM:    33       33      C        5.1870   -4.5270   -2.1140
  QMMM:    34       34      C        4.1680   -4.9560   -2.9420
  QMMM:    35       35      C        6.5810   -4.7490   -0.0610
  QMMM:    36       36      H        0.0660   -6.5820   -7.3390
  QMMM:    37       37      H        1.1720   -5.7590   -8.4130
  QMMM:    38       38      H       -0.1830   -5.3390  -10.2650
  QMMM:    39       39      H       -2.0720   -4.0810  -11.2250
  QMMM:    40       40      H       -3.3190   -2.9580   -7.3010
  QMMM:    41       41      H       -1.4360   -4.2130   -6.3370
  QMMM:    42       42      H       -4.5720   -2.2990   -9.0790
  QMMM:    43       43      H       -4.5570   -3.3290  -10.5070
  QMMM:    44       44      H        1.0960   -3.2030   -7.4960
  QMMM:    45       45      H        3.5040   -4.3520   -6.2050
  QMMM:    46       46      H        3.4690   -2.6520   -6.6560
  QMMM:    47       47      H        4.7520   -5.5010   -7.7230
  QMMM:    48       48      H        5.2210   -6.1640  -10.0440
  QMMM:    49       49      H        4.0740   -5.0370  -11.8960
  QMMM:    50       50      H        2.4540   -3.2300  -11.4490
  QMMM:    51       51      H        1.9750   -2.5640   -9.1190
  QMMM:    52       52      H       -0.0260   -2.7100   -5.4500
  QMMM:    53       53      H        2.6640   -2.9460   -4.1560
  QMMM:    54       54      H        1.5640   -4.3580   -2.5670
  QMMM:    55       55      H       -0.7760   -4.8540   -4.3140
  QMMM:    56       56      H       -0.8110   -3.5600   -3.1280
  QMMM:    57       57      H       -1.9070   -6.6170   -3.6520
  QMMM:    58       58      H       -2.5010   -8.3190   -1.9840
  QMMM:    59       59      H       -1.6190   -8.1550    0.2910
  QMMM:    60       60      H       -0.1400   -6.2760    0.9290
  QMMM:    61       61      H        0.4600   -4.5620   -0.7580
  QMMM:    62       62      H        2.7510   -7.2550   -4.1990
  QMMM:    63       63      H        1.6110   -7.0410   -2.8950
  QMMM:    64       64      H        3.1300   -7.6400   -1.1760
  QMMM:    65       65      H        4.9500   -6.8750    0.3010
  QMMM:    66       66      H        5.7640   -3.6520   -2.3750
  QMMM:    67       67      H        3.9510   -4.4140   -3.8540
  QMMM:    68       68      H        7.1560   -3.9960   -0.5860
  QMMM:    69       69      H        7.2210   -5.5810    0.1780
  QMMM:    70       70      H       -4.2200   -1.2400  -11.1740
  QMMM:    71       71      H        0.7540   -1.0590   -6.5080
  QMMM:    72       72      H        1.7120   -2.0870   -2.4090
  QMMM:    73       73      H        5.6000   -4.9090    1.6140
  QMMM:    74       74      N        1.0850   -4.9920   -6.4930
  QMMM:    75       75      N        1.6610   -5.5010   -4.2700
  QMMM:    76       76      O        1.4170   -7.0440   -5.8040
  QMMM:    77       77      O       -3.4810   -1.5960  -10.6770
  QMMM:    78       78      O        1.4010   -1.3640   -5.8670
  QMMM:    79       79      O        1.1760   -1.9950   -3.2010
  QMMM:    80       80      O        6.0510   -4.2050    1.1410

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -112.3161 kcal/mol        0.7167 kcal/(mol*A)
xmin    20     -113.3894 kcal/mol        0.6712 kcal/(mol*A)
xmin    30     -115.6503 kcal/mol        0.6207 kcal/(mol*A)
xmin    40     -117.2251 kcal/mol        0.2798 kcal/(mol*A)
xmin    50     -117.3729 kcal/mol        0.3960 kcal/(mol*A)
xmin    60     -117.4352 kcal/mol        0.0666 kcal/(mol*A)
xmin    70     -117.6994 kcal/mol        0.2379 kcal/(mol*A)
xmin    80     -117.8331 kcal/mol        0.0955 kcal/(mol*A)
xmin    90     -117.9253 kcal/mol        0.2132 kcal/(mol*A)
xmin   100     -117.9950 kcal/mol        0.0757 kcal/(mol*A)
xmin   110     -118.0064 kcal/mol        0.0471 kcal/(mol*A)
xmin   120     -118.0349 kcal/mol        0.1385 kcal/(mol*A)
xmin   130     -118.0581 kcal/mol        0.0636 kcal/(mol*A)
xmin   140     -118.0788 kcal/mol        0.0284 kcal/(mol*A)
xmin   150     -118.0987 kcal/mol        0.0747 kcal/(mol*A)
xmin   160     -118.1251 kcal/mol        0.1173 kcal/(mol*A)
xmin   170     -118.1726 kcal/mol        0.1403 kcal/(mol*A)
xmin   180     -118.2716 kcal/mol        0.2274 kcal/(mol*A)
xmin   190     -118.3480 kcal/mol        0.1263 kcal/(mol*A)
xmin   200     -118.3614 kcal/mol        0.0194 kcal/(mol*A)
xmin   210     -118.3670 kcal/mol        0.0521 kcal/(mol*A)
xmin   220     -118.3682 kcal/mol        0.0088 kcal/(mol*A)
xmin   230     -118.3687 kcal/mol        0.0185 kcal/(mol*A)
xmin   240     -118.3693 kcal/mol        0.0059 kcal/(mol*A)
xmin   250     -118.3697 kcal/mol        0.0030 kcal/(mol*A)
xmin   260     -118.3698 kcal/mol        0.0040 kcal/(mol*A)
xmin   270     -118.3698 kcal/mol        0.0010 kcal/(mol*A)
xmin   280     -118.3698 kcal/mol        0.0012 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -41.7111  -40.2474  -40.0310  -39.7178  -39.3354  -39.2527  -38.2674  -37.8016
 -37.5576  -36.5315  -36.2610  -34.0580  -33.9476  -32.8074  -32.3446  -31.6249
 -31.5540  -31.4471  -31.4225  -31.1105  -29.7744  -28.6070  -27.5933  -27.4895
 -26.0455  -25.4757  -24.6447  -24.1935  -23.5549  -23.4740  -23.2443  -23.2249
 -22.3369  -22.1086  -21.7505  -21.5844  -21.0447  -19.8541  -18.7757  -18.5498
 -18.5014  -18.3212  -17.9116  -17.4721  -17.3778  -17.1754  -17.1506  -17.0350
 -16.9340  -16.5622  -16.3538  -16.2907  -16.1821  -16.1032  -15.9696  -15.9271
 -15.7633  -15.7393  -15.6640  -15.4671  -15.3006  -15.1492  -14.8413  -14.7609
 -14.6458  -14.4501  -14.3079  -14.2594  -14.1735  -14.1380  -14.0351  -13.9639
 -13.8660  -13.7519  -13.5950  -13.4833  -13.3319  -13.2292  -13.1085  -13.0831
 -12.9855  -12.7963  -12.6170  -12.6002  -12.5052  -12.3095  -12.2185  -12.0839
 -12.0778  -11.9201  -11.8770  -11.7713  -11.7444  -11.6794  -11.4618  -11.3766
 -11.1595  -11.0540  -10.9335  -10.1067   -9.9044   -9.8203   -9.7825   -9.7019
  -9.6834   -9.5686   -9.4174   -9.3666   -9.2331    0.0764    0.1884    0.3867
   0.4112    0.4134    0.4623    0.5226    0.5320    1.2538    1.3431    2.3442
   2.6996    2.7240    2.7782    2.8098    2.9476    3.1796    3.2200    3.2415
   3.2898    3.5628    3.6352    3.6377    3.6725    3.7164    3.7583    3.7987
   3.8109    3.8198    3.8468    3.8656    3.8947    3.9184    3.9335    3.9664
   3.9953    4.0130    4.0221    4.0352    4.0454    4.0874    4.0941    4.1165
   4.1627    4.1740    4.1944    4.2615    4.2929    4.3233    4.3663    4.3931
   4.4091    4.4615    4.4844    4.5190    4.5325    4.5397    4.5876    4.6184
   4.6330    4.6817    4.6862    4.6954    4.7768    4.9112    4.9296    4.9973
   5.0671    5.1043    5.1164    5.1329    5.1407    5.2442    5.3027    5.3423
   5.3821    5.4694    5.5015    5.5471    5.5580    5.5587    5.5913    5.6108
   5.6150    5.6745    5.7162    5.7191    5.8263    5.9294    5.9644    6.0431
   6.0871    6.0936    6.2193    6.2953    6.3879    7.0732

 Heat of formation   =       -118.36985496 kcal/mol  (       -5.13290208 eV)

 Total SCF energy    =    -161888.04593718 kcal/mol  (    -7019.99245207 eV)
 Electronic energy   =   -1697360.17411146 kcal/mol  (   -73603.06032312 eV)
 Core-core repulsion =    1535472.12817429 kcal/mol  (    66583.06787105 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                  0.386
     2      C                 -0.009
     3      C                 -0.088
     4      C                 -0.128
     5      C                 -0.105
     6      C                 -0.079
     7      C                 -0.135
     8      C                 -0.112
     9      C                  0.018
    10      C                 -0.009
    11      C                 -0.156
    12      C                 -0.083
    13      C                 -0.129
    14      C                 -0.129
    15      C                 -0.131
    16      C                 -0.132
    17      C                 -0.129
    18      C                 -0.010
    19      C                  0.023
    20      C                  0.054
    21      C                 -0.143
    22      C                 -0.073
    23      C                 -0.123
    24      C                 -0.129
    25      C                 -0.132
    26      C                 -0.130
    27      C                 -0.133
    28      C                  0.017
    29      C                 -0.101
    30      C                 -0.117
    31      C                 -0.131
    32      C                 -0.114
    33      C                 -0.113
    34      C                 -0.115
    35      C                  0.015
    36      H                  0.124
    37      H                  0.092
    38      H                  0.138
    39      H                  0.154
    40      H                  0.131
    41      H                  0.137
    42      H                  0.071
    43      H                  0.073
    44      H                  0.112
    45      H                  0.104
    46      H                  0.116
    47      H                  0.154
    48      H                  0.135
    49      H                  0.131
    50      H                  0.131
    51      H                  0.130
    52      H                  0.104
    53      H                  0.099
    54      H                  0.119
    55      H                  0.106
    56      H                  0.117
    57      H                  0.133
    58      H                  0.132
    59      H                  0.131
    60      H                  0.131
    61      H                  0.136
    62      H                  0.133
    63      H                  0.100
    64      H                  0.133
    65      H                  0.128
    66      H                  0.138
    67      H                  0.152
    68      H                  0.111
    69      H                  0.072
    70      H                  0.203
    71      H                  0.212
    72      H                  0.225
    73      H                  0.199
    74      N                 -0.300
    75      N                 -0.316
    76      O                 -0.408
    77      O                 -0.326
    78      O                 -0.340
    79      O                 -0.334
    80      O                 -0.320
 Total Mulliken Charge =       0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -3.352    0.593   -0.513    3.443
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.4822   -5.8072   -5.2479
  QMMM:     2        2      C        0.4994   -5.5096   -7.4506
  QMMM:     3        3      C       -0.7026   -4.8619   -8.0755
  QMMM:     4        4      C       -0.5924   -4.2229   -9.3135
  QMMM:     5        5      C       -1.7171   -3.6617   -9.9173
  QMMM:     6        6      C       -2.9606   -3.7272   -9.2836
  QMMM:     7        7      C       -3.0702   -4.3610   -8.0393
  QMMM:     8        8      C       -1.9481   -4.9259   -7.4403
  QMMM:     9        9      C       -4.1906   -3.1530   -9.9054
  QMMM:    10       10      C        1.4101   -3.5594   -6.2586
  QMMM:    11       11      C        2.9488   -3.4633   -6.2643
  QMMM:    12       12      C        3.5386   -3.8153   -7.5839
  QMMM:    13       13      C        3.8750   -5.1424   -7.8803
  QMMM:    14       14      C        4.4569   -5.4620   -9.1057
  QMMM:    15       15      C        4.7080   -4.4617  -10.0451
  QMMM:    16       16      C        4.3745   -3.1389   -9.7558
  QMMM:    17       17      C        3.7936   -2.8159   -8.5305
  QMMM:    18       18      C        0.8250   -2.6511   -5.1472
  QMMM:    19       19      C        1.3021   -3.0328   -3.7421
  QMMM:    20       20      C        0.7842   -4.4159   -3.2650
  QMMM:    21       21      C       -0.7526   -4.4771   -3.3581
  QMMM:    22       22      C       -1.3231   -5.5432   -2.4912
  QMMM:    23       23      C       -1.8801   -6.6969   -3.0546
  QMMM:    24       24      C       -2.4314   -7.6831   -2.2374
  QMMM:    25       25      C       -2.4338   -7.5244   -0.8521
  QMMM:    26       26      C       -1.8839   -6.3746   -0.2846
  QMMM:    27       27      C       -1.3317   -5.3882   -1.0989
  QMMM:    28       28      C        2.1628   -6.4530   -2.9947
  QMMM:    29       29      C        3.5668   -6.0402   -2.6507
  QMMM:    30       30      C        3.8975   -5.6722   -1.3434
  QMMM:    31       31      C        5.2099   -5.3298   -1.0254
  QMMM:    32       32      C        6.2032   -5.3422   -2.0106
  QMMM:    33       33      C        5.8692   -5.7051   -3.3190
  QMMM:    34       34      C        4.5584   -6.0540   -3.6372
  QMMM:    35       35      C        7.6124   -4.9888   -1.6576
  QMMM:    36       36      H        0.2633   -6.5994   -7.2695
  QMMM:    37       37      H        1.3729   -5.4678   -8.1667
  QMMM:    38       38      H        0.3837   -4.1659   -9.8192
  QMMM:    39       39      H       -1.6314   -3.1590  -10.8938
  QMMM:    40       40      H       -4.0464   -4.4128   -7.5348
  QMMM:    41       41      H       -2.0358   -5.4231   -6.4618
  QMMM:    42       42      H       -4.7634   -2.5534   -9.1444
  QMMM:    43       43      H       -4.8482   -3.9880  -10.2769
  QMMM:    44       44      H        1.0466   -3.1205   -7.2410
  QMMM:    45       45      H        3.3856   -4.1282   -5.4700
  QMMM:    46       46      H        3.2220   -2.4042   -6.0007
  QMMM:    47       47      H        3.6680   -5.9362   -7.1413
  QMMM:    48       48      H        4.7188   -6.5066   -9.3299
  QMMM:    49       49      H        5.1695   -4.7153  -11.0104
  QMMM:    50       50      H        4.5732   -2.3474  -10.4931
  QMMM:    51       51      H        3.5404   -1.7697   -8.3027
  QMMM:    52       52      H       -0.3010   -2.6795   -5.1877
  QMMM:    53       53      H        2.4292   -3.0122   -3.7157
  QMMM:    54       54      H        1.0371   -4.4440   -2.1557
  QMMM:    55       55      H       -1.0864   -4.6268   -4.4181
  QMMM:    56       56      H       -1.1375   -3.4762   -3.0156
  QMMM:    57       57      H       -1.8809   -6.8265   -4.1475
  QMMM:    58       58      H       -2.8650   -8.5878   -2.6884
  QMMM:    59       59      H       -2.8696   -8.3023   -0.2087
  QMMM:    60       60      H       -1.8884   -6.2445    0.8075
  QMMM:    61       61      H       -0.9051   -4.4777   -0.6511
  QMMM:    62       62      H        2.2049   -7.4689   -3.4907
  QMMM:    63       63      H        1.5550   -6.5574   -2.0494
  QMMM:    64       64      H        3.1223   -5.6555   -0.5625
  QMMM:    65       65      H        5.4678   -5.0511    0.0074
  QMMM:    66       66      H        6.6435   -5.7140   -4.1012
  QMMM:    67       67      H        4.2916   -6.3478   -4.6667
  QMMM:    68       68      H        8.2928   -5.0744   -2.5488
  QMMM:    69       69      H        7.9787   -5.6782   -0.8464
  QMMM:    70       70      H       -4.6489   -1.9938  -11.3741
  QMMM:    71       71      H        0.8786   -1.0074   -6.1745
  QMMM:    72       72      H        1.0646   -1.2233   -3.0991
  QMMM:    73       73      H        7.0863   -3.4748   -0.5661
  QMMM:    74       74      N        0.8575   -4.9029   -6.1723
  QMMM:    75       75      N        1.4728   -5.5308   -3.8780
  QMMM:    76       76      O        1.9899   -6.8808   -5.6611
  QMMM:    77       77      O       -3.8238   -2.3143  -10.9917
  QMMM:    78       78      O        1.2613   -1.3098   -5.3428
  QMMM:    79       79      O        0.8054   -2.1017   -2.7891
  QMMM:    80       80      O        7.7610   -3.6460   -1.2342

           --------- Calculation Completed ----------

